modules/ding_event/ding_event.module

```diff
@@ -227,3 +227,24 @@ function ding_event_preprocess_views_view(&$vars) {
     }
   }
 }
+
+/**
+ * Implements hook_permission().
+ */
+function ding_event_permission() {
+  return array(
+    'administer event settings' => array(
+      'title' => t('Administer event settings'),
+    ),
+  );
+}
+
+/**
+ * Implements hook_views_pre_build().
+ */
+function ding_event_views_pre_build(&$view) {
+  if ($view->name == "ding_event" && $view->current_display == 'ding_event_list_frontpage') {
+    $view_items_per_page = $view->display['ding_event_list_frontpage']->display_options['pager']['options']['items_per_page'];
+    $view->items_per_page = variable_get('ding_event_frontend_items_per_page', $view_items_per_page);
```

Comment:
Will this force the exported view / feature to be listed as overridden when `ding_event_frontend_items_per_page` is set? If yes: Is there any way to avoid that from happening while keeping the desired result?